doc/reference/reference_lua/net_box.rst

```diff
@@ -162,55 +164,79 @@ Below is a list of all ``net.box`` functions.
-      :ref:`console.connect() <console-connect>` instead.
-
-    * `connect_timeout`: number of seconds to wait before returning "error: Connection timed out".
+    *   ``reconnect_after``: if ``reconnect_after`` is greater than zero, then a ``net.box`` instance will try 
+        to reconnect if a connection is broken or if a connection attempt fails. This makes transient network 
+        failures become transparent to the application. Reconnect happens automatically in the background, 
+        so requests that initially fail due to connectivity loss are transparently retried. The number of 
+        retries is unlimited, connection attempts are made after each specified interval (for example ``reconnect_after=5`` 
+        means try to reconnect every 5 seconds). When a connection is explicitly closed, or when the Lua garbage collector 
+        removes it, then reconnect attempts stop. The default value of ``reconnect_after``, as with other ``connect`` 
+        options, is ``nil``.
+    *   ``call_16``: [since 1.7.2] by default, ``net.box`` connections comply with a new binary protocol command for CALL, 
+        which is not backward compatible with previous versions. The new CALL no longer restricts a function to returning 
+        an array of tuples and allows returning an arbitrary MsgPack/JSON result, including scalars, nil and void (nothing). 
+        The old CALL is left intact for backward compatibility. It will be removed in the next major release. All programming 
+        language drivers will be gradually changed to use the new CALL. To connect to a Tarantool instance that uses the old 
+        CALL, specify ``call_16=true``.
+    *   ``console``: depending on the option's value, the connection supports different methods (as if instances of different 
+        classes were returned). With ``console = true``, you can use ``conn`` methods ``close()``, ``is_connected()``, 
+        ``wait_state()``, ``eval()`` (in this case, both binary and Lua console network protocols are supported). With 
+        ``console = false`` (default), you can also use ``conn`` database methods (in this case, only the binary protocol is 
+        supported). Deprecation notice: ``console = true`` is deprecated, users should use :ref:`console.connect() <console-connect>` instead.
+    *   ``connect_timeout``: number of seconds to wait before returning "error: Connection timed out".
+    *   ``required_protocol_version``: depending on the value of the option, the connection requires the minimum version 
+        of the IPROTO protocol supported by the server. If the server version is lower than specified, the connection will 
+        fail with an error message. With ``required_protocol_version = 1`` all connections fail where the server protocol is 
+        lower than ``1``.  
+    *   ``required_protocol_features``: depending on the value of the option, the connection requires the specified 
```

Comment:
The same comment as for the previous option: why not to start with the straight definition.

Reply:
Fixed.